src/averageAges.js

```diff
@@ -51,8 +68,24 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const woman = people.filter(child =>
+    (!onlyWithSon)
```

Comment:
```suggestion
    !onlyWithSon 
```

no need of the parentheses